src/mongo/db/storage/record_store.h

```diff
@@ -360,6 +360,15 @@ class RecordStore {
         return true;
     }
 
+    /**
+     * Relaxed version of findRecord which allows to implement optimistic locking strategy.
```

Comment:
The second half of this comment is incorrect.  It should read more like:
"....which allows storage engines to skip a pessimistic locking strategy."